app/components/admin/users/tools/emails_component.html.erb

```diff
@@ -0,0 +1,74 @@
+<section
+  class="crayons-card p-3 js-component"
+  data-controller="users--tools--ajax"
+  data-action="ajax:success@document->users--tools--ajax#success ajax:error@document->users--tools--ajax#error">
+  <%# Used to announce to the screen reader client side which section the user is in %>
+  <div id="section-title" class="hidden">Emails section</div>
+
+  <h3>
+    <%= link_to "← Tools", admin_user_tools_path(@user.id),
+                remote: true,
+                data: { action: "ajax:success@document->user#replacePartial" },
+                class: "crayons-title fs-l" %>
```

Comment:
It's a minor thing and can be swept up later, but it would be good if this link had a focus style (or really, all links in the admin section - I don't think it's specific to here 😅 )

Reply:
Let me add that to all the back links!